LunaMary/src/QuizLibrary.java

```diff
@@ -0,0 +1,40 @@
+import java.util.List;
+
+public class QuizLibrary {
+
+
+    private List<Quiz> quizList;
+    private List<AnswerPerson> answerPeople;
+    private String type;
+
+    public QuizLibrary(List<Quiz> quizList, List<AnswerPerson> answerPeople, String type) {
```

Comment:
You should have an empty constructor.
And then a few methods that let you add more Quizzes to the quizList.